tests/output.rs

```diff
@@ -67,9 +67,24 @@ fn strip_indent<'line>(line: &'line str, indent: &str) -> &'line str {
     &line[indent.len()..]
 }
 
+#[cfg(target_os = "macos")]
+const DYNAMIC_LIBRARY_EXTENSION: &str = ".dylib";
+#[cfg(all(unix, not(target_os = "macos")))]
```

Comment:
These sort of seemed like the correct way to specify these? Let's see if CI passes for Linux/Windows (I was testing locally on Mac and that seemed to be working)

Reply:
Ah interesting, so I guess BSD, etc fall under this too (not just Linux)?

Reply:
Right and hopefully they all also use `.so` (at least if they want to run this test suite successfully)